lstchain/calib/camera/pedestals.py

```diff
@@ -34,13 +34,13 @@ class PedestalIntegrator(PedestalCalculator):
 
      """
     charge_median_cut_outliers = List(
-        [-3, 3],
+        [-4, 4],
```

Comment:
why is this changed 3->4?

Reply:
Even if we assume that all pixels follow a common distribution, cutting an interval of 3 sigma will always remove ~5 pixels erroneously. That's way to harsh.

With 4 sigma, the expected number of erroneously rejected pixels is 0.1:
```
In [10]: def expected_rejected_pixels(a, b):
    ...:     return (1 - (norm.cdf(b) - norm.cdf(a))) * 1855
    ...: 

In [11]: expected_rejected_pixels(-3, 3)
Out[11]: 5.008121697347684

In [12]: expected_rejected_pixels(-4, 4)
Out[12]: 0.11750030720087512
```